test/e2e/engine/template.go

```diff
@@ -243,6 +243,20 @@ func (e *Engine) HasNetworkPolicy(name string) bool {
 	return false
 }
 
+// HasAllZonesAgentPools will return true if all of the agent pools have zones
+func (e *Engine) HasAllZonesAgentPools() bool {
```

Comment:
missing unit test here

Reply:
Do you mean e2e tests? https://github.com/Azure/acs-engine/blob/7ab8b9529084b8cafa698cd46247b29d74fce722/test/e2e/kubernetes/kubernetes_test.go#L852-L934 
Do we have unit tests for `test/e2e/engine/template.go`

Reply:
Sorry you're right we don't currently have unit tests for it since it's in the test package, please disregard my comment.